src/modules/menu/components/activeLayers/MenuActiveLayersListItemTimeSelector.vue

```diff
@@ -129,8 +129,16 @@ function hidePopover() {
                 :key="timeEntry.timestamp"
                 class="btn mb-1 me-1"
                 :class="{
-                    'btn-primary': timeEntry.timestamp === timeConfig.currentTimestamp,
-                    'btn-light': timeEntry.timestamp !== timeConfig.currentTimestamp,
+                    'btn-primary': isTimeSliderActive
+                        ? previewYear === timeEntry.year
+                        : timeConfig.currentTimestamp === timeEntry.timestamp,
+                    'btn-outline-primary':
+                        isTimeSliderActive &&
+                        timeConfig.currentTimestamp === timeEntry.timestamp &&
+                        timeEntry.year !== previewYear,
+                    'btn-light':
+                        timeEntry.timestamp !== timeConfig.currentTimestamp &&
+                        (!isTimeSliderActive || previewYear !== timeEntry.year),
```

Comment:
This should be simplified with a computed value for each 3 use cases. Something like this

```javascript
'btn-primary': isSelected,
'btn-outline-primary': isSecondarySelected,
'btn-light': !isSelecte && !isSecondarySelected
```

However I'm not sure anymore did we not talked that the timeslider would also changes the time configuration of all visible layers with time config instead of a preview ?

Reply:
Hey, I discussed this quickly with Pascal. we did indeed decide to alter the behavior of the time slider to have it change the years to all layers, but this change was not reflected in the tickets written after the discussion. We're keeping this for the post go live.

We made a [ticket](https://jira.swisstopo.ch/browse/PB-411) to discuss what we want to do with the time slider after the go live. Changing the time of layers is one of them, but we had a few suggestions and ideas that could be nice to discuss and refine.

On another note : the computed approach would make this easier to read. I'll change it.